<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTorrentParamsWidget</class>
 <widget class="QWidget" name="AddTorrentParamsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="managementLayout">
     <item>
      <widget class="QLabel" name="labelTorrentManagementMode">
       <property name="text">
        <string>Torrent Management Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboTTM">
       <property name="toolTip">
        <string>Automatic mode means that various torrent properties(eg save path) will be decided by the associated category</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSavePath">
     <property name="title">
      <string>Save at</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="defaultsNoteLabel">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Note: the current defaults are displayed for reference.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FileSystemPathLineEdit" name="savePathEdit" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="downloadPathLayout">
        <item>
         <widget class="QLabel" name="useDownloadPathLabel">
          <property name="text">
           <string>Use another path for incomplete torrents:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="useDownloadPathComboBox"/>
        </item>
        <item>
         <spacer name="downloadPathSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="FileSystemPathLineEdit" name="downloadPathEdit" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="categoryLayout" stretch="0,2,1">
     <item>
      <widget class="QLabel" name="labelCategory">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="categoryComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertPolicy::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="categorySpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tagsLayout" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="tagsLabel">
       <property name="text">
        <string>Tags:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tagsLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Click [...] button to add/remove tags.</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tagsEditButton">
       <property name="toolTip">
        <string>Add/remove tags</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="miscParamsWidget" native="true">
     <widget class="QWidget" name="startTorrentWidget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>65</y>
        <width>143</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="startTorrentLabel">
         <property name="text">
          <string>Start torrent:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="startTorrentComboBox">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="contentLayoutWidget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>159</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="contentLayoutLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="contentLayoutLabel">
         <property name="text">
          <string>Content layout:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="contentLayoutComboBox">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stopConditionWidget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>93</y>
        <width>158</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="stopConditionLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="stopConditionLabel">
         <property name="text">
          <string>Stop condition:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stopConditionComboBox">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="addToQueueTopWidget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>121</y>
        <width>187</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="addToQueueTopLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="addToQueueTopLabel">
         <property name="text">
          <string>Add to top of queue:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="addToQueueTopComboBox">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="skipCheckingCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>83</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Skip hash check</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="torrentShareLimitsBox">
     <property name="title">
      <string>Torrent share limits</string>
     </property>
     <layout class="QVBoxLayout" name="torrentShareLimitsBoxLayout">
      <item>
       <widget class="TorrentShareLimitsWidget" name="torrentShareLimitsWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSystemPathLineEdit</class>
   <extends>QWidget</extends>
   <header>gui/fspathedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TorrentShareLimitsWidget</class>
   <extends>QWidget</extends>
   <header>gui/torrentsharelimitswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
